docs/en/status-communities/import-a-status-community.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 445
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Import a Status Community
```

Comment:
The "Import community" button is not easy for mobile users to see, users may not know they can import a community. Hence, they may not want to read "import a community" or search for it. 
How about changing the title to something like "View a community that's shared with you"

Reply:
Thanks, @cheny0 - I'm rewriting and restructuring the content based on your feedback. 

It wasn't easy to find an article's name. The one you proposed makes sense, but it's a passive sentence, which I prefer to avoid for titles. At the same time, I want to avoid using the "Join" word because the user is not joining the community yet; it's only adding/importing it. The "join" part comes later, as long as s/he is accepted and meets the community requirements.

I ended up with "Import a community someone shares with you".